components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +107,17 @@ export function StartPage(props: StartPageProps) {
                 )}
                 {error && <StartError error={error} />}
                 {props.children}
+                {props.showLatestIdeWarning && (
+                    <InfoBox className="mt-4">
+                        You are using the latest editor version.{" "}
```

Comment:
@gtsiolis  Maybe the failure or STOP is caused by the latest IDE? In that case, we should display this

Reply:
@mustard-mh Correct! But these are different workspace lifecycle phases, right? For example if the workspace crashes it's an error state.

**question:** Can the workspace go to the normal `STOPPING` and `STOPPED` phases just because of using the latest?

Reply:
> question: Can the workspace go to the normal STOPPING and STOPPED phases just because of using the latest?

Yes, some issues (like this https://github.com/gitpod-io/gitpod/pull/8526) may stop workspace in seconds, so users cannot see that error message